.github/workflows/pytest-workflow.yml

```diff
@@ -56,12 +56,12 @@ jobs:
             tags: concatenate_vcfs
           - profile: "conda"
             tags: sentieon/bwamem
+          - profile: "singularity"
+            tags: concatenate_vcfs
           - profile: "singularity"
             tags: merge
           - profile: "singularity"
             tags: validation_checks
-          - profile: "conda"
-            tags: validation_checks
```

Comment:
Ah, sorry. I thought it was a duplicate.

Reply:
could be, but this PR and this branch are for sentieon related stuff only, as we're not going to merge this to dev right away, most likely in the next release rather than the coming one, so if you find any issues or inconsistencies, or fix anything not sentieon related, please make a PR for it to `dev` and sync your branch to get the update in your code.
(convoluted I know, but this is a huge feature)
And you're already doing an amazing job